### ISSUE TITLE
NetworkBehaviour: Awake calls Base instead of Derived

### ISSUE THREAD

**Issue by xKamuna** [FirstGearGames/FishNet#32]
Base code:
```
    protected virtual void Awake()
    {
        Debug.Log("Base called.");
    }
 ```
 Derived code:
 ```
    protected override void Awake()
    {
        Debug.Log("Derived called.");
    }
 ```
 Expected Output: Derived called.
 Actual Output: Base called.

**Comment by FirstGearGames** [comment]
I actually have notes in the codegen to change this behavior, for that exact reason. You can expect an update soon.

**Comment by FirstGearGames** [comment]
@xKamuna This has been resolved in 1.4.7.
Base calls have been moved out of awake and put into the networkInitialize methods (created by codegen)
`  protected override void Awake()
  {
    this.NetworkInitialize___Early();
    this.NetworkInitialize__Late();
  }

  public override void NetworkInitialize___Early() => base.NetworkInitialize___Early();

  public override void NetworkInitialize__Late() => base.NetworkInitialize__Late();`
